### PR TITLE
add extra gopath and resources

### PR DESCRIPTION
**Pull request by lunny**
If we have more than one resources directory

### CODE REVIEW

File: bundler.go [hunk 1]
```diff
@@ -48,6 +48,8 @@ type Configuration struct {
 	// The path of the go binary
 	// Best is to leave it empty. Default value is "go"
 	GoBinaryPath string `json:"go_binary_path"`
+	// The path of the resouces, default is resouces
```
Comment (asticode) [inline bundler.go]: Default should be `resources` not `resouces`

File: bundler.go [hunk 2]
```diff
@@ -150,8 +152,16 @@ func New(c *Configuration) (b *Bundler, err error) {
 	}
 
 	// Paths that depends on the input path
-	b.pathBuild = strings.TrimPrefix(strings.TrimPrefix(b.pathInput, filepath.Join(os.Getenv("GOPATH"), "src")), string(os.PathSeparator))
-	b.pathResources = filepath.Join(b.pathInput, "resources")
+	if strings.HasPrefix(b.pathInput, filepath.Join(os.Getenv("GOPATH"), "src")) {
```
Comment (asticode) [inline bundler.go]: Why did you add this `if` condition? If this is to allow using relative path and avoid the `TrimPrefix(os.PathSeparator)`, I'd rather you use `filepath.IsAbs()` method or event better the `filepath.Abs()` method

File: bundler.go [hunk 3]
```diff
@@ -150,8 +152,16 @@ func New(c *Configuration) (b *Bundler, err error) {
...
+		b.pathBuild = strings.TrimPrefix(strings.TrimPrefix(b.pathInput, filepath.Join(os.Getenv("GOPATH"), "src")), string(os.PathSeparator))
+	}
+
+	if len(c.ResourcesPath) == 0 {
+		c.ResourcesPath = []string{"resouces"}
```
Comment (asticode) [inline bundler.go]: Default value should be `resources` instead of `resouces`

Comment (asticode) [inline bundler.go]: And it should be `filepath.Join(b.pathInput, "resources")` not just `resources`

File: bundler.go (hunk 2 above)
Comment (lunny) [reply bundler.go]: For gop, a folder is a temp `GOPATH`, so the build path is relative to this temp `GOPATH`. This `if` will confirm the required build package is in the global `GOPATH`.

File: bundler.go (hunk 1 above)
Comment (lunny) [reply bundler.go]: done.

File: bundler.go (hunk 3 above)
Comment (lunny) [reply bundler.go]: done.

Comment (lunny) [reply bundler.go]: see line 162.

File: bundler.go (hunk 2 above)
Comment (asticode) [reply bundler.go]: ok but you're doing `if has prefix then trim prefix` which is the same as `trim prefix`. In this case I'd rather you remove the if condition. Or maybe there's something I'm missing out.

**Comment by asticode** [comment]
@lunny did you have time to see my last comment in the review ?

```ok but you're doing if has prefix then trim prefix which is the same as trim prefix. In this case I'd rather you remove the if condition. Or maybe there's something I'm missing out.```
